src/index.ts

```diff
@@ -80,15 +80,40 @@ export const runSass = function (
     sassOpts.loadPaths = [sassPath];
```

Comment:
I went back and forth on whether to remove this injection of True into `loadPaths` to allow `@use 'true';`. On one hand it can easily be replaced by `@use 'pkg:sass-true' as true;` -- on the other hand it doesn't hurt anything, and makes it easier for people using slightly older versions of Dart Sass to use True. I ended up leaving it around, partially because it makes internal testing much simpler.